### ISSUE TITLE
Block Models with Issues

### ISSUE THREAD

**Issue by alexhliu** [chunky-dev/chunky#888]
(Will update as I find more block issues)

- [ ] Comparator is missing the bottom
- [ ] Iron door top is incorrectly in shadow sometimes

![image](https://user-images.githubusercontent.com/42661490/114457465-e980e080-9b92-11eb-8ac2-75e0201ed704.png)
![image](https://user-images.githubusercontent.com/42661490/114457636-1a611580-9b93-11eb-8fe3-71162d510d46.png)

**Comment by alexhliu** [comment]
Minecraft:
![2021-04-12_13 36 16](https://user-images.githubusercontent.com/42661490/114458629-48932500-9b94-11eb-9a55-6dab93b86016.png)
Chunky:
![image](https://user-images.githubusercontent.com/42661490/114458662-547ee700-9b94-11eb-8333-0a7837035015.png)

**Comment by StanleyMines** [comment]
The second one is showing farmland I assume?

**Comment by alexhliu** [comment]
Yes, second one is farmland. (The textures are correct, its just the model that is wrong)

**Comment by alexhliu** [comment]
![image](https://user-images.githubusercontent.com/42661490/114506729-0c3ce480-9be7-11eb-8026-ec5bf14a82e6.png)

**Comment by alexhliu** [comment]
![image](https://user-images.githubusercontent.com/42661490/114510612-b6b70680-9beb-11eb-9182-507195a4b388.png)

**Comment by alexhliu** [comment]
![image](https://user-images.githubusercontent.com/42661490/114625404-0edf1e80-9c67-11eb-81ce-bcf3893ad8bf.png)

**Comment by alexhliu** [comment]
![Uploading image.png…]()

**Comment by leMaik** [comment]
![image](https://user-images.githubusercontent.com/5544859/117888258-1ddede00-b2b2-11eb-91ce-6ba6a7b09de0.png)
(the sideways connecting is already fixed but the quads are clipping)

**Comment by leMaik** [comment]
- Comparators, repeaters and iron doors have been fixed.
- Pressure plate, observer and tripwire fix is on the way ⌛ 
- I can't reproduce the chorus clipping issues anymore. 🤔
